Dixit_Service/Dixit_Service.csproj

```diff
@@ -75,7 +86,7 @@
     <VisualStudio>
       <FlavorProperties GUID="{3D9AD99F-2412-4246-B90B-4EAA41C64699}">
         <WcfProjectProperties>
-          <AutoStart>True</AutoStart>
+          <AutoStart>False</AutoStart>
```

Comment:
ok, it was failing because of the service contract, but it is resolved in another PR